.pipelines/3p-e2e.yml

```diff
@@ -0,0 +1,87 @@
+
+        stages:
+            - stage: e2e_test
+              displayName: "3P E2E Tests"
+              jobs:
+                  - template: .pipelines/templates/msal-browser-e2e.yml@1P
+                    parameters:
+                        jobName: "validate_msal_browser"
+                        poolType: "linux"
+                        publishFlagName: "publishMsalBrowser"
+                        sourceBranchName: ${{ variables.sourceBranchName }}
+                        stage: "PR"
+                        override: true
+                        samples:
+                            - "client-capabilities"
+                            - "onPageLoad"
+                            - "pop"
+                            - "customizable-e2e-test"
+                  - template: .pipelines/templates/msal-node-e2e.yml@1P
+                    parameters:
+                        jobName: "validate_msal_node"
+                        poolType: "windows"
+                        publishFlagName: "publishMsalNode"
+                        sourceBranchName: ${{ variables.sourceBranchName }}
+                        stage: "PR"
+                        override: true
+                        node: [16, 18, 20, 22]
+                        samples:
+                            - "auth-code"
+                            - "auth-code-cli-app"
+                            - "client-credentials"
+                            - "device-code"
+                            - "silent-flow"
+                            - "b2c-user-flows"
+                            # - "on-behalf-of"
+                  - template: .pipelines/templates/msal-react-e2e.yml@1P
+                    parameters:
+                        jobName: "validate_msal_react"
+                        poolType: "linux"
+                        publishFlagName: "publishMsalReact"
+                        sourceBranchName: ${{ variables.sourceBranchName }}
+                        stage: "PR"
+                        override: true
+                        samples:
+                            - "nextjs-sample"
+                            - "react-router-sample"
+                            - "gatsby-sample"
+                            - "typescript-sample"
+                            - "b2c-sample"
+                  - template: .pipelines/templates/msal-angular-e2e.yml@1P
+                    parameters:
+                        jobName: "validate_msal_angular"
+                        poolType: "linux"
+                        publishFlagName: "publishMsalAngular"
+                        sourceBranchName: ${{ variables.sourceBranchName }}
+                        stage: "PR"
+                        override: true
+                        samples:
+                            - "angular15-sample-app"
+                            - "angular16-sample-app"
+                            - "angular-b2c-sample-app"
+                            - "angular-standalone-sample"
```

Comment:
Were you going to add `angular17-standalone-sample` here too?

Reply:
Done